src/drivers/serial/imx_uart/imx_uart.c

```diff
@@ -17,7 +17,7 @@ EMBOX_UNIT_INIT(uart_init);
 
 #define IOMUXC_BASE	OPTION_GET(NUMBER,iomuxc_base)
 #define UART_NUM	OPTION_GET(NUMBER,num)
-#define IRQ_NUM		(58 + UART_NUM)
+#define IRQ_NUM		((32 + 26) + UART_NUM)
```

Comment:
Why this is neccesary?

Reply:
for greater clarity. from an documentation irq number 26+uart_num

Reply:
It's not clear what are `32` and `26`, so it's better to declare it as a macro.